src/azure/cli/commands/arm.py

```diff
@@ -172,32 +172,39 @@ def handler(args):
                       if key in get_arguments}
         instance = getter(client, **getterargs) if client else getter(**getterargs)
 
+        for k in args.copy().keys():
+            if k in get_arguments or k in set_arguments \
+                or k in ('properties_to_add', 'properties_to_remove', 'properties_to_set'):
+                args.pop(k)
+        for key, val in args.items():
+            ordered_arguments.append((key, val))
```

Comment:
ordered args previously only took the set, add and remove params.  Now we add additional params so they will be processed by the custom handlers

---
In reply to: [74598948](https://github.com/Azure/azure-cli/pull/687#discussion_r74598948) [](ancestors = 74598948)